### PR TITLE
Update extension screen styles

### PR DESCRIPTION
**Pull request by mikejolley**
Makes everything full width to match the search bar, cleans up spacing, and adds flexbox to the products grid so they resize responsively.

Featured:

woocommerce extensions local wordpress dev wordpress 2017-12-08 13-34-26

Grid:

woocommerce extensions local wordpress dev wordpress 2017-12-08 13-34-46

### CODE REVIEW

**Comment by belcherj** [comment]
screen shot 2017-12-08 at 8 39 57 am

On very large screens, think 27", it gets a little wonky.  I'm cool with it if you are.

**Comment by garymurray** [comment]
@belcherj @mikejolley the 27 inch and bigger issue is my concern - so I would rather just constrain it to the 1140px width if we want to ship something with the beta then.

**Comment by belcherj** [comment]
My screenshot was a zoom out, I am not in front of a 27", I will test later today when I get in front of my monitor.

File: assets/css/admin.scss [hunk 1]
```diff
@@ -65,7 +66,7 @@
 	}
 
 	.addons-featured {
-		max-width: 1140px;
+		margin: 0 1em 0 0;
```
Comment (belcherj) [inline assets/css/admin.scss]: this margin is off from the rest of the page: 
screen shot 2017-12-08 at 8 56 45 am

I may need to fix the search bar.

File: assets/css/admin.scss [hunk 2]
```diff
@@ -194,17 +193,21 @@
 	}
 
 	.addons-column {
-		flex: 1 1 200px;
-		margin: 0 1%;
-		width: 49.5%;
+		flex: 1;
+		margin-right: 1em;
```
Comment (belcherj) [inline assets/css/admin.scss]: screen shot 2017-12-08 at 8 59 07 am

This breaks mobile styling once everything goes vertical.

**Review by belcherj — CHANGES_REQUESTED** [review]

**Comment by mikejolley** [comment]
Can we constrain the entire wrapper including search form then?

**Review by belcherj — APPROVED** [review]
LGTM!

**Pull request merged**